### PR TITLE
Pr locosys m300

### PR DESCRIPTION
**Pull request by farhangnaderi**
This is a setup I have used from LOCOSYS RTK station and realized that the community may be able to benefit from it. The whole bundle can be purchased with an affordable price compared to most other available RTK sets with the prices starting from approximately 5k $. 

@hamishwillee please let me know how we can proceed publishing it. The setup for PX4 is still experimental by the way and need some work which I am working on it.

### CODE REVIEW

File: en/gps_compass/rtk_gps_locosys_r2.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Locosys RTK-compass GPS
```
Comment (hamishwillee) [inline en/gps_compass/rtk_gps_locosys_r2.md:1]: This is already in docs - remove http://docs.px4.io/main/en/gps_compass/rtk_gps_locosys_r2.html

File: rtk_gps_locosys_r1.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+# Freefly Systems RTK GPS
```
Comment (hamishwillee) [inline rtk_gps_locosys_r1.md:1]: This is already in docs so remove - see http://docs.px4.io/main/en/gps_compass/rtk_gps_locosys_r1.html

File: en/SUMMARY.md [hunk 3]
```diff
@@ -240,6 +240,7 @@
       * [Holybro H-RTK-F9P](gps_compass/rtk_gps_holybro_h-rtk-f9p.md)
       * [Holybro H-RTK-M8P](gps_compass/rtk_gps_holybro_h-rtk-m8p.md)
       * [Locosys Hawk R1](gps_compass/rtk_gps_locosys_r1.md)
+      * [Locosys Hawk R2](gps_compass/rtk_gps_locosys_r1.md)
```
Comment (hamishwillee) [inline en/SUMMARY.md:243]: This is a duplicate line - I think you want to add "Locosys M300 RTK system"?

File: en/gps_compass/rtk_gps.md [hunk 4]
```diff
@@ -31,6 +31,7 @@ GPS | Yaw Output | [Dual F9P GPS Heading](../gps_compass/u-blox_f9p_heading.md)
 [Holybro H-RTK F9P Rover Lite](../gps_compass/rtk_gps_holybro_h-rtk-f9p.md) | | | |
 [Holybro H-RTK M8P GNSS](../gps_compass/rtk_gps_holybro_h-rtk-m8p.md) | | | |
 [LOCOSYS Hawk R1](../gps_compass/rtk_gps_locosys_r1.md) | | | |
+[LOCOSYS Hawk R2](../gps_compass/rtk_gps_locosys_r2.md) |&check; | | |
```
Comment (hamishwillee) [inline en/gps_compass/rtk_gps.md:34]: This line is OK. Did you also want to add Locosys M300 RTK system  here?

File: en/gps_compass/rtk_gps_locosys_m300.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+LOCOSYS RTK-M300 supports global GPS/Glonass/Beidou/Galileo/QZSS satellites , L1+L5 Dual-frequency and multi-constellation RTK positioning solution.
+RTK-M300 provides full frequency 4G-LTE, UMTS/HSPA+ and GSM/GPRS/EDGE coverage.
+It features 10/100/1000Mbps Ethernet data and voice connectivity.
+It provides convenient SIM access via an external SIM socket
+RTK-M300 installs Win10 (or Linux) operating system, suitable for with LOCOSYS Firebird application software,provide user-friendly graphical operation interface, whether it is used for “Base station” management or “Rover” use.
+
+Whether as a RTK Base station or RTK Rover, due to its fan-less and compact design it is eaasy to install and use.
+In particular, it is suitable for base stations where space is at a premium.
+RTK-M300 keeps the flexibility to meet different demands for telemetric monitor or surveying applications.
```
Comment (hamishwillee) [inline en/gps_compass/rtk_gps_locosys_m300.md:16]: This is a bit of a mess - you are trying to say too much. I gave it a subedit but it still needs more work.

File: en/gps_compass/rtk_gps_locosys_m300.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
+# Locosys M300 RTK system 
+
+[LOCOSYS RTK-M300](https://www.locosystech.com/en/product/RTK-System/rtk-M300.html) is a compact and rugged advanced RTK (Real-time kinematic) receiver and industrial grade computer with Intel Atom® x5-E3930 processor dual core 1.3GHz (boost up to 1.8GHz).
+It is designed for harsh conditions, or where you need a noiseless ad-hoc network environment.
+
+![LOCOSYS RTK-M300 mini pc](../../assets/hardware/gps/locosys_m300/locosys_m300_general.png)
+
+LOCOSYS RTK-M300 supports global GPS/Glonass/Beidou/Galileo/QZSS satellites , L1+L5 Dual-frequency and multi-constellation RTK positioning solution.
+RTK-M300 provides full frequency 4G-LTE, UMTS/HSPA+ and GSM/GPRS/EDGE coverage.
```
Comment (hamishwillee) [inline en/gps_compass/rtk_gps_locosys_m300.md:9]: Do you mean that this supports telephony with all these bands? If you have a SIM installed of course?

**Comment by hamishwillee** [comment]
@farhangnaderi What do you mean by "Experimental"? I.e. does it work? What is still happening? If it isn't ready for use yet, then I'd rather not merge it.

**Comment by hamishwillee** [comment]
@farhangnaderi There are some questions for you on this. I can help further subedit, but we need removal of the clashing updates.

**Comment by farhangnaderi** [comment]
> @farhangnaderi There are some questions for you on this. I can help further subedit, but we need removal of the clashing updates.

Hamish thanks a bunch for reviewing this. I need to figure the experimental thing out as they have different setups and there are two ways (serial or Ntrip server) to make it work. Ntrip server one is actually a QGC PR now and not merged. 
I just am confused if we can put this temporarily until  resolve the setup issues so that it is ready for user or we make it a draft only till everything is settled on QGC and PX4 side for this specific setup?

**Comment by hamishwillee** [comment]
Hi @farhangnaderi 
Sounds good. I've made it draft until we have a clearer picture. Ping me when you think this is in a good useable state for end users.

**Comment by farhangnaderi** [comment]
> Hi @farhangnaderi
> Sounds good. I've made it draft until we have a clearer picture. Ping me when you think this is in a good useable state for end users.

Thanks! I will take care of it!

**Comment by hamishwillee** [comment]
@farhangnaderi I'm closing this as "too out of date". You can recreate when you're ready :-)

**Comment by farhangnaderi** [comment]
Thanks @hamishwillee . My bad I forgot to close this!

**Pull request closed without merging**